### ISSUE TITLE
Getting error - HarRecorder is not defined

### ISSUE THREAD

**Issue by qburst-hari** [loadmill/har-recorder#1]
Can we use it in sync mode. When I add the below step -

const { startRecording, endRecording } = HarRecorder();

I am getting error  - HarRecorder is not defined

**Comment by idoco** [comment]
Hi @qburst-hari can you share some additional information about this issue? (Maybe a code example?)
When you run this Selenium test without the har-recorder does it successfully load this HTML page?

**Comment by qburst-hari** [comment]
Yes, I am using webdriverio selenium for automation. so since by default chrome driver listens to port 4444. I changed the debugging port also to port 4444. When I run the selenium test without har-recorder,  my automation page loads. But when I add har recorder. It loads this selenium page.

**Comment by idoco** [comment]
Interesting. Can you try to clone and run this [example project](https://github.com/loadmill/selenium-demo#how-to-run-this) and let me know if it works for you?

I'm trying to understand if this is a system configuration issue or a problem in the way the recorder is used in your code.

**Comment by qburst-hari** [comment]
Sure, let me check and update you

**Comment by qburst-hari** [comment]
Thats working for me. There are some errors, but the har file is generated. for me the problem is. I am starting my selenium standalone in port 4444. so for the chrome debug mode, If I add any other port - I will get error - connect ECONNREFUSED 127.0.0.1:9223

If I add 4444. I am getting that html page which I dont have any clue what it does.

**Comment by qburst-hari** [comment]
My chrome configuration is like below : 

      browserName: "chrome",
      applicationName: "common",
      "goog:chromeOptions": {
        mobileEmulation: { deviceName: "Pixel 2" },
        w3c: false,
        args: [
          "--remote-debugging-port=9223",
          "--reduce-security-for-testing",
          "--use-mobile-user-agent",
          "--user-agent=Mozilla/5.0 (iPhone; CPU iPhone OS 12_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) CriOS/79.0.3945.73 Mobile/15E148 Safari/605.1",
          "--window-size=0,870"
        ]
      }

**Comment by qburst-hari** [comment]
var HarRecorder = require("har-recorder")
const { startRecording, endRecording } = HarRecorder();

 onPrepare: function() {
    return startRecording({ port: 9223 })
  },

onComplete: function() {
    return endRecording('ABBA.har');
},

I have used it with hooks of webdriverio

**Comment by idoco** [comment]
This looks like it should work 🤔 
Do you have a full example that I can clone and run locally? I'd really like to try and solve this :)

**Comment by idoco** [comment]
Maybe the webdriverio hooks work differently than my selenium-webdriver example?
Maybe you should await the startup of the Chrome instance or before you start the recording?

**Comment by harismahesh** [comment]
Hi, I have  invited you as a collabarator for a sample project. Please have a look.

**Comment by idoco** [comment]
Thanks @harismahesh I will try to run this tomorrow

**Comment by qburst-hari** [comment]
Hi @idoco  got any chance to look into the files

**Comment by harismahesh** [comment]
Hi @idoco , Also another issue I faced is I am not able to run testcases in parallel.
